scripts/update_docs.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+
+set -x
+
+SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+DIR=$(cd $SCRIPTS_DIR; cd ..; pwd)
+VERSION=$1
+TOC_NAME=$(echo ${VERSION}toc|sed s/\\./-/g)
+
+read -r -d '' CONFIG_VERSION_BLOCK << EOM
+latest: v0.14.3
```

Comment:
This is the hardcoded version that I mentioned. It should be changed to work for the release after this one.

Reply:
Fixed. It was as easy as using `v.*`. I'm guessing those `.` in the existing line were already acting as wildcards unintentionally but they just happened to match the literal `.` when perl was doing its matching.